src/reducers/entities/teams.js

```diff
@@ -22,6 +22,7 @@ function teams(state = {}, action) {
     case TeamTypes.RECEIVED_TEAMS_LIST:
         return Object.assign({}, state, teamListToMap(action.data));
     case TeamTypes.RECEIVED_TEAMS:
+    case SchemeTypes.RECEIVED_SCHEME_TEAMS:
         return Object.assign({}, state, action.data);
```

Comment:
Isn't `action.data` an array for `RECEIVED_SCHEME_TEAMS`? Probably needs to be moved up with `RECEIVED_TEAMS_LIST`

Reply:
blargh